Query.php

```diff
@@ -246,7 +246,9 @@ public function all($db = null)
         if ($this->emulateExecution) {
             return [];
         }
-        $result = $this->createCommand($db)->search();
+        $result = $this->createCommand($db)->search([
+            'track_scores' => 'true'
```

Comment:
This is no good solution - pls don't hard code your own one single case. Tracking scores adds extra overhead which may not be OK for others. You already have `$options` that might be supplied along into query e.g. `$result = $this->createCommand($db)->search($this->options)` or add a new flag e.g. `public $trackScores = false`. Unit test?

Reply:
I agree, let's not hardcode it.